src/components/MessageList/MessageListInner.js

```diff
@@ -51,7 +51,12 @@ const insertDates = (messages, lastRead, userID) => {
     }
 
     if (!unread) {
-      unread = lastRead && lastRead.getTime() < message.created_at.getTime();
+      if (typeof lastRead === 'string') {
+        unread =
+          lastRead && Date.parse(lastRead) < message.created_at.getTime();
+      } else {
+        unread = lastRead && lastRead.getTime() < message.created_at.getTime();
+      }
```

Comment:
you think sth like this works?
```suggestion
      unread = lastRead && new Date(lastRead) < message.created_at;
```